clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/CatsSearchProvider.groovy

```diff
@@ -173,22 +173,26 @@ class CatsSearchProvider implements SearchProvider {
       log.info("no query string specified, looked for sensible default and found: ${q}")
     }
 
-    log.info("Querying ${toQuery} for term: ${q}")
+    log.info("Querying ${cachesToQuery} for term: ${q}")
     String normalizedWord = q.toLowerCase()
-    List<String> matches = new ArrayList<String>()
-    toQuery.each { String cache ->
-      matches.addAll(cacheView.filterIdentifiers(cache, "*:${cache}:*${normalizedWord}*").findAll { String key ->
-        try {
-
-          // if the key represented in the cache doesn't actually exist, don't process it
-          if (keyProcessors && !keyProcessors.any { it.canProcess(cache) && it.exists(key) }) {
```

Comment:
This line was causing the server groups and load balancers to disappear from search.  There is one `keyProcessor` and it's for `serverGroups`.  When the caches for, e.g., `securityGroups` were checked, this line returned false because no `keyProcessors` said that it exists (because there are no key processors for security groups, etc).